### PR TITLE
Better docs

### PR DESCRIPTION
**Pull request by GrosQuildu**

### CODE REVIEW

File: docs/fuzzing.md [hunk 1]
```diff
@@ -130,6 +152,22 @@ Dirs:
 * PULL_DIR  - out/the_fuzzer/queue
 * CRASH_DIR - out/the_fuzzer/crashes
 
+Synchronization:
+* AFL executor runs the fuzzer in auto-sync mode (`-M`)
+* Test cases pushed to PUSH_DIR will be automatically used by the AFL
```
Comment (disconnect3d) [inline docs/fuzzing.md]:
```suggestion
* Test cases pushed to `PUSH_DIR` will be automatically used by the AFL
```

File: docs/fuzzing.md [hunk 2]
```diff
@@ -130,6 +152,22 @@ Dirs:
 * PULL_DIR  - out/the_fuzzer/queue
 * CRASH_DIR - out/the_fuzzer/crashes
 
+Synchronization:
```
Comment (disconnect3d) [inline docs/fuzzing.md]: Is it describing what happens under the hood or what the user has to do?

In the first case, we might rename this e.g. `Actions done by DeepState for synchronization:` (?) or highlight that its internal info(?)

Comment (disconnect3d) [reply docs/fuzzing.md]: same below

File: docs/fuzzing.md [hunk 3]
```diff
@@ -148,6 +186,19 @@ Dirs:
...
+* libFuzzer executor runs the fuzzer in auto-sync mode (`-reload=1`)
+* Test cases pushed to PUSH_DIR will be automatically used by the libFuzzer
+* Filenames may be arbitrary
+
+Resuming:
+* libFuzzer uses test cases from each specified dir,
+so the executor only needs to use PULL_DIR to resume fuzzing
```
Comment (disconnect3d) [inline docs/fuzzing.md]:
```suggestion
so the executor only needs to use `PULL_DIR` to resume fuzzing
```
same in other places and for `PUSH_DIR`

File: docs/test_harness.md [hunk 4]
```diff
@@ -1,5 +1,279 @@
...
+}
+```
+
+
+## Symbolic variables - inputs
+
+Executors need to know what variables are symbolic,
+that is which should be monitored and used by them in order to explore
+application state space. Symbolic variables will be used
+as an unknowns in equations during symbolic execution or populated
```
Comment (disconnect3d) [inline docs/test_harness.md]:
```suggestion
as unknowns in equations during symbolic execution or populated
```

File: docs/fuzzing.md (hunk 2 above)
Comment (GrosQuildu) [reply docs/fuzzing.md]: Its mix of both: sometimes I write that AFL do smthing and sometimes that executor do smthing. But thats more like internal/advanced info. Will describe that a bit more maybe

**Pull request merged**